tests/functional/aws-node-sdk/test/bucket/deleteBucketReplication.js

```diff
@@ -81,7 +81,7 @@ describe('aws-node-sdk test deleteBucketReplication', () => {
             }),
             next => deleteReplicationAndCheckResponse(bucket, next),
             next => s3.getBucketReplication({ Bucket: bucket }, err => {
-                assert(errors.ReplicationConfigurationNotFoundError[err.code]);
+                assert(errors.ReplicationConfigurationNotFoundError.is[err.code]);
```

Comment:
```suggestion
                assert(errors.ReplicationConfigurationNotFoundError.is[err.type]);
```

`code` is the http status code

Reply:
Alternatively, this is just `err.is.ReplicationConfigurationNotFoundError`, unless this error comes from another component and/or wasn't rebuilt into an ArsenalError.